jenkins/build.sh

```diff
@@ -15,22 +15,27 @@
 # about this software or licensing, please email opensource@seagate.com or
 # cortx-questions@seagate.com.
 
-
 set -ex
 BASE_DIR=$(realpath "$(dirname $0)/..")
 PROG_NAME=$(basename $0)
 DIST=$(realpath $BASE_DIR/dist)
 RPM_NAME="cortx-ha"
 CORTX="cortx"
 HA_PATH="/opt/seagate/${CORTX}"
+HA1="1"
+HA2="2"
+BUILD_VERSION="1"
+MINOR_VERSION="0.0"
 
 usage() {
     echo """
-usage: $PROG_NAME [-v <coretx-ha version>] [-d]
-                            [-b <build no>] [-k <key>]
+usage: $PROG_NAME [-g <cortx-ha git version>] [-v <cortx-ha major version>] [-m <cortx-ha minor version>]
```

Comment:
This is signature of build.sh in Ujjwal's e-mail -
# build.sh [<major version>] [<minor version>] [<revision>] [<build>]

major=${1:MY_MAJOR_VERSION}
minor=${2:MY_MINOR_VERSION}
revision=${3:MY_REVISON_VERSION}
build=${4:MY_BUILD_VERSION}

Reply:
Done